RULES.md

```diff
@@ -848,7 +848,7 @@ Such stops normally do not provide user value. This notice may indicate a typo i
 
 #### StopWithoutZoneIdNotice
 
-Per GTFS specification, `stops.zone_id` should be provided if fare information is provided using `fare_rules.txt`. This rule does not apply to records from `stops.txt` that represent a stop or en entrance (i.e. `stops.location_type = 1 or 2`).
+Per GTFS specification, `stops.zone_id` should be provided if fare information is provided using `fare_rules.txt`. This rule does not apply to records from `stops.txt` that represent a station or en entrance (i.e. `stops.location_type = 1 or 2`).
```

Comment:
"a station or en entrance" has a typo (en entrance) and looks outdated.

Reply:
Thanks, fixed in https://github.com/MobilityData/gtfs-validator/pull/972/commits/9789068e732890bc5eede91e2ca82dcd574d5cf0 and https://github.com/MobilityData/gtfs-validator/pull/972/commits/f836516e35f3f3bc711683039a1bff7930477bf9.